### PR TITLE
docs: Added DocsGPT integration

### PR DESCRIPTION
**Pull request by Anush008**
[Preview](https://deploy-preview-657--condescending-goldwasser-91acf0.netlify.app/documentation/frameworks/docsgpt/).

### CODE REVIEW

File: qdrant-landing/content/documentation/frameworks/docsgpt.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: DocsGPT
+weight: 2600
+---
+
+# DocsGPT
+
+[DocsGPT](https://docsgpt.arc53.com/) is an open-source documentation assistant that enables you to build conversational user experiences on top of your data.
+
+Qdrant is supported as vectorstore in DocsGPT to ingest and semantically retrieve documents.
```
Comment (mjang) [inline qdrant-landing/content/documentation/frameworks/docsgpt.md:10]:
```suggestion
Qdrant is supported as a vectorstore in DocsGPT to ingest and semantically retrieve documents.
```

**Review by mjang — CHANGES_REQUESTED** [review]
Mostly minor changes. One question on where readers would find definitions for the list of `QDRANT_*` environment variables.

File: qdrant-landing/content/documentation/frameworks/docsgpt.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+DocsGPT configurations are applied using environment variables in a `.env` file.
+
+To configure DocsGPT to use Qdrant as the vector store, set `VECTOR_STORE` to `"qdrant"`.
+
+```bash
+echo "VECTOR_STORE=qdrant" >> .env
+```
+
+Find the list of all the Qdrant configuration options that can be configured via environment variables [here](https://github.com/arc53/DocsGPT/blob/00dfb07b15602319bddb95089e3dab05fac56240/application/core/settings.py#L46-L59).
```
[the remark this answers is not in the document]
Comment (Anush008) [reply qdrant-landing/content/documentation/frameworks/docsgpt.md:24]: Will do.

**Comment by Anush008** [comment]
> One question on where readers would find definitions for the list of `QDRANT_*` environment variables.

Currently, there isn't a section in the DocsGPT doc that explains the possible env values. 
There's one small section here.
https://docs.docsgpt.co.uk/Guides/How-to-use-different-LLM#step-1-configure-environment-variables

I'll PR one as you suggested.

**Review by mjang — APPROVED** [review]

**Pull request merged**